### PR TITLE
[JS Bundles] Outbound call composite bundle

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Introduce new JS bundle to use the StartCall API in the CallAdapter instead of the locator like the existing bundle
# Why

Make it so that we are able to pass the id's of the participants that we are calling directly to composite without needing to do a map from the old participants locator

Part of WIKA asks
# How Tested

Built and Validated locally

### CODE REVIEW

File: samples/StaticHtmlComposites/outboundCallComposite.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <!-- Ensure that fonts have loaded before tests are run -->
+  <link rel="preload" href="./fonts/segoeui-regular.woff2" as="font" crossorigin="anonymous" />
+  <link rel="preload" href="./fonts/segoeui-semibold.woff2" as="font" crossorigin="anonymous" />
+  <link rel="preload" href="./fonts/segoeui-bold.woff2" as="font" crossorigin="anonymous" />
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+  <div id="outbound-call-composite-container"></div>
+  <!-- replace with https://github.com/Azure/communication-ui-library/releases/latest/download/callComposite.js for development and prototyping -->
```
Comment (PorterNan) [inline samples/StaticHtmlComposites/outboundCallComposite.html:16]:
```suggestion
  <!-- replace with https://github.com/Azure/communication-ui-library/releases/latest/download/outboundCallComposite.js for development and prototyping -->
```

File: samples/StaticHtmlComposites/src/outboundCallComposite.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    userId,
+    displayName: displayName ?? 'anonymous',
+    credential: new AzureCommunicationTokenCredential(token),
+    targetCallees: targetCallees || [
+      targetCallees.foreach((callee) => {
+        return fromFlatCommunicationIdentifier(callee);
+      })
+    ]
+  });
+  console.log('adapter', adapter);
```
Comment (PorterNan) [inline samples/StaticHtmlComposites/src/outboundCallComposite.js:27]: Remove console.log

File: samples/StaticHtmlComposites/src/outboundCallComposite.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import {
+  CallComposite,
+  createAzureCommunicationCallAdapter,
+  fromFlatCommunicationIdentifier
+} from '@azure/communication-react';
+import { initializeIcons } from '@fluentui/react';
+initializeIcons();
+export const loadCallComposite = async function (args, htmlElement, props) {
+  const { userId, token, displayName, targetCallees } = args;
+  console.log('targetCallees', targetCallees);
```
Comment (PorterNan) [inline samples/StaticHtmlComposites/src/outboundCallComposite.js:16]: Remove

**Review by PorterNan — APPROVED** [review]

File: samples/StaticHtmlComposites/outboundCallComposite.html (hunk 1 above)
Comment (dmceachernmsft) [reply samples/StaticHtmlComposites/outboundCallComposite.html:16]: ah I guess part of this is adding it to the CI workflow to host it on github, I will do that as well!

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**